README.md

```diff
@@ -1,6 +1,7 @@
 # Maps API for JavaScript
 
-This repository holds a series of JavaScript based examples using the **HERE Maps API for JavaScript**. More information about the API can be found on [developer.here.com](https://developer.here.com/develop/javascript-api) under the JavaScript APIs section.
+This repository holds a series of JavaScript based examples using the **HERE Maps API for JavaScript**. More information about the API can be found on [developer.here.com](https://developer.here.com/develop/javascript-api) under the JavaScript APIs section. 
+To run the examples, clone this repo to a folder on your Desktop. Replace the credentials in the **test-credentials.js** file with your own credentials.
```

Comment:
I would avoid giving direct instructions on where to clone the repo.
Therefore I would rephrase this like so:


```suggestion
To run the examples, clone this repo and replace the credentials in the **test-credentials.js** file with your own credentials.
```